.github/workflows/testAndPublish.yml

```
@@ -0,0 +1,268 @@
+name: Run tests and publish
```

This name is quite verbose, particularly when reading the lists of sub jobs e.g. "Run tests and publish / Create launcher". Thoughts @SaschaCowley?

```suggestion
name: CI/CD
```